### PR TITLE
Implementation of all repositories of a user from user->settings

### PR DESCRIPTION
**Pull request by DblK**
This PR implement the display of all repository owned by the user in its settings.

Here is the UI:

The menu:
image

With no repository:
image

With all kind of repositories:
image

### CODE REVIEW

File: options/locale/locale_en-US.ini
Comment (sondr3) [inline options/locale/locale_en-US.ini]: The phrasing on this is a bit awkward, "You do not own any repositories" is more correct. Although I'm not sure using "own" here is the right term either.

Comment (DblK) [reply options/locale/locale_en-US.ini]: "You do not have any repositories", maybe?

Comment (sondr3) [reply options/locale/locale_en-US.ini]: Ah, yeah, that sounds a lot better in my opinion. Maybe even "You do not currently have any repositories". 😃

File: options/locale/locale_en-US.ini
Comment (pgaskin) [inline options/locale/locale_en-US.ini]: That's a bit wordy and is not consistent with the rest of the strings. I think it should be "You do not own any repositories".

Comment (sondr3) [reply options/locale/locale_en-US.ini]: I commented on this earlier and I don't like the wording where you own repositories, I get the meaning behind it but don't think it's too accurate.

Comment (pgaskin) [reply options/locale/locale_en-US.ini]: All the other strings refer to it as owning repositories. Consistency makes everything easier to use and look like everything fits together. What do you mean by it isn't too accurate.

File: options/locale/locale_fr-FR.ini
Comment (bkcsoft) [inline options/locale/locale_fr-FR.ini]: This should be done via CrowdIn. (all languages other than English)

**Review by bkcsoft — CHANGES_REQUESTED** [review]
Otherwise LGTM :)

File: options/locale/locale_fr-FR.ini
Comment (DblK) [reply options/locale/locale_fr-FR.ini]: I removed the french translation

**Review by bkcsoft — APPROVED** [review]

**Comment by bkcsoft** [comment]
Looks good 🙂  why the ad-hoc list though? It differs from all other lists (tables) in the App. Looks very "thrown together" and out of place 😕

**Comment by DblK** [comment]
You are right. I didn't know how to display it. It was the same as Github was before.
Here is the new UI I've done for this:
image

L-G-T-M does not seems to see your last comments.
Commit and push will be done pretty soon ;)

**Comment by strk** [comment]
LGTM - but it would be great if you could add integration tests for this new endpoint.

**Comment by pgaskin** [comment]
LGTM

**Comment by lunny** [comment]
@geek1011 so many conflicted files?

**Comment by pgaskin** [comment]
@lunny I was saying L-G-T-M about the idea, and the fact that it works when that branch is built, not the conflicts when merging it.

**Comment by bkcsoft** [comment]
fixed merge-conflicts

**Comment by lunny** [comment]
It seems there is a line break after the icon?
untitled

**Comment by lunny** [comment]
@DblK Please confirm the problem, maybe we could move this to v1.2.

**Comment by pgaskin** [comment]
@lunny @DblK Yes, there is a problem. 

It can be fixed by making the icon and content inline-block, instead of floating it. Floating is less predictable, and it does not work because the content div fills as much space as possible.

**Comment by DblK** [comment]
I'll fix it soon I hope kinda busy right now.

**Comment by lunny** [comment]
This could be merged after the little problem resolved. @DblK maybe some maintainer could help you if you are busy now.

**Comment by DblK** [comment]
If someone else cqn finish it go for it. Not much time right now.

**Comment by lunny** [comment]
rebased and see below screenshot.

**Pull request merged**
